identity/identity.proto

```diff
@@ -200,6 +200,27 @@ message Capability {
     Type type = 1;
   }
 
+  // EncryptionKeyRotation contains the features of the EncryptionKeyRotation
+  // operation that the CSI-driver supports.
+  message EncryptionKeyRotation {
+    // Type describes a CSI Service that CSI-drivers can support.
```

Comment:
`Type describes a CSI Service that CSI-drivers can support` to `Type describes a CSI Encryption Service that CSI driver can support`

Reply:
Ack.